yarn.lock

```diff
@@ -7662,9 +7708,9 @@ react-transition-group@^1.2.0:
     warning "^3.0.0"
 
 react@^16.4.2, react@^16.9.0:
-  version "16.9.0"
-  resolved "https://registry.yarnpkg.com/react/-/react-16.9.0.tgz#40ba2f9af13bc1a38d75dbf2f4359a5185c4f7aa"
-  integrity sha512-+7LQnFBwkiw+BobzOF6N//BdoNw0ouwmSJTEm9cglOOmsg/TMiFHZLe2sEoN5M7LgJTj9oHH0gxklfnQe66S1w==
+  version "16.12.0"
```

Comment:
I am assuming this is a safe upgrade... I'm not seeing much in the way of release notes from React after 16.9, so hopefully...

Reply:
Yeah, really ought to be fine...